### DISCUSSION TITLE
Warning messages in Create Program component

### DISCUSSION THREAD

**Discussion by petrasvestartas** [visose/Robots#91] [Q&A] [answered]
Thank you for the Robots plugin. 
I am very new to this plugin. 

I would like to know about a few warning messages in **Create Program** component.

In the current simulation I have only 3 planes, and I get these warning messages:
_Configuration changed to "None" on target 1 of robot 0
Configuration changed to "Wrist" on target 2 of robot 0_

What do they mean and I what should I do to make this component run without warnings?

![image](https://user-images.githubusercontent.com/18013985/82064427-071ca000-96cd-11ea-8c03-85d42ffcc66f.png)

**Comment by visose** [comment, marked as answer]
Hi, you can check on the wiki the Target > Configuration and Motion type sections.
In summary, in a joint motion where you don't specify a configuration, it will choose the closest one (the one where the joints rotate the least). If the configuration chosen is different from the previous target it will give you a warning, just in case. A warning is not an error, its more like, double check you want this, it might not be what you expect. For example, if it adopts a configuration with the elbow bending in the opposite direction, it might collide with the environment. Linear motions always remain on the same configuration as the previous target.
